
syntax = "proto3";

package downloadshort;
option go_package = "."; 
service DownloadShort {
  rpc DownTiktok (ParamsRequest) returns (ReturnsReply) {}
  rpc DownYoutube (ParamsRequest) returns (ReturnsReply) {}

}

message ParamsRequest {
  string url = 1;
}

message ReturnsReply {
  string status = 1;
}
